cmd/servicelog/cmd.go

```diff
@@ -0,0 +1,23 @@
+package servicelog
+
+import (
+	"github.com/spf13/cobra"
+)
+
+//func NewCmdServiceLog(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *cobra.Command {
+func NewCmdServiceLog() *cobra.Command {
+	// servicelogCmd represents the servicelog command
```

Comment:
this is straightforward in the code, so I don't think it's needed

Reply:
solved